File: src/main/java/org/example/EjerJava/Ej2.java
package org.example.EjerJava;


import java.util.Scanner;



public class Ej2 {
    public static void main(String[] args) {
        System.out.print("Introduce un número: ");
        String respuesta  = new Scanner(System.in).nextLine();
        int numero = Integer.parseInt(respuesta);

        if (numero % 2 == 0){
            System.out.println("El número es par.");
        }else {
            System.out.println("El número no es par.");
        }
    }
}

File: src/main/java/org/example/ejerKotlin/Ej2.kt
package org.example.ejerKotlin

fun main() {
    print("Introduzca un número: ")
    val num = readln().toInt()

    if (num % 2 == 0) {
        println("El numero $num es par")
    }else{
        println("El numero $num es impar")
    }
}